src/rgw/driver/sfs/object.cc

```diff
@@ -54,16 +54,13 @@ int SFSObject::SFSReadOp::get_attr(const DoutPrefixProvider *dpp,
                                                  const char *name,
                                                  bufferlist &dest,
                                                  optional_yield y) {
-  ldpp_dout(dpp, 10) << __func__ << ": TODO: " << name << dendl;
-
-  if (std::strcmp(name, "user.rgw.acl") == 0) {
-    // TODO support 'user.rgw.acl' to support read_permissions. Return
-    // empty policy since our test user is admin for now.
-    RGWAccessControlPolicy policy;
-    policy.encode(dest);
-    return 0;
+  if (!objref || objref->deleted) {
+    return -ENOENT;
   }
-  return -ENOTSUP;
+  if(!objref->get_attr(name, dest)){
+    return -ENODATA;
+  }
```

Comment:
there's inconsistent formatting throughout the patchset when it comes to spacing. I understand this is a personal preference thing, but if so it would be nice to keep the same throughout the patchset. Even though there are coding guidelines for the Ceph project (which we have been very good at not following) to which we should aim adhering to.

Reply:
I agree with you that we should try to follow the same convention (and I'm trying) and in this case I will fix this (because you point me out).
Unfortunately, this is something I'm not good at all without an automatic tool that is doing this kind of work for me.
I will always do mistakes of this kind (This is why I tend to rely completely on formatting tools).
So expect my code to be potentially inconsistent from time to time.

Reply:
If you find a formatting tool that allows us to just format code that has been edited, that would be awesome. We could define the rules for everybody, and have it go through the checks to ensure code conformity. We just can't be doing that for the entire code base :\